python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -226,7 +226,7 @@ def get_schema(cls):
                       minimum: 0
                   - type: number
                     minimum: 0
-                default: 5
+                default: [5,  5, 5, 5, 5, 15, 15, 30, 30, 30, 30,  30, 30, 30, 30,  30, 30, 30, 30,  30, 30, 30, 30]
```

Comment:
This is throwing a flake8 error in the CI testing.

Reply:
Fixed, and version history updated.  I'm 100% on board with having a test pass before merging.